react-shopping-cart/src/App.js

```diff
@@ -0,0 +1,60 @@
+import Header from "./component/@shared/Header/Header";
+import PageTitle from "./component/@shared/PageTitle/PageTitle";
+import { ReactComponent as Cart } from "./assets/cart.svg";
+import { ColumnFlexWrapper, RowFlexWrapper } from "./styles/Wrapper";
+import NavigateButton from "./component/@shared/NavigateButton/NavigateButton";
+import ProductListPage from "./pages/ProductListPage/ProductListPage";
+import { Routes, Route, useNavigate } from "react-router-dom";
+import ProductDetailPage from "./pages/ProductDetailPage/ProductDetailPage";
+import ShoppingCartPage from "./pages/ShoppingCartPage/ShoppingCartPage";
+
+function App() {
+  const navigate = useNavigate();
+
+  const handleNavigateHome = () => {
+    navigate("/");
+  };
+
+  const handleNavigateShoppingCart = () => {
+    navigate("/shopping-cart");
+  };
+
+  return (
+    <>
+      <Header>
+        <PageTitle onClick={handleNavigateHome}>
+          <Cart />
+          <div>WOOWA SHOP</div>
+        </PageTitle>
+        <RowFlexWrapper gap="20px">
+          <NavigateButton onClick={handleNavigateShoppingCart}>
```

Comment:
`PageTitle` 도 `NavigateButton`로 부터 나온 컴포넌트인 것 같은데요. 이름과 달리 스타일을 제외한 아무런 역할이 없는 `NavigateButton`이 아쉽습니다 이름만 보면 쉽게 이동을 시켜줄 수 있는 컴포넌트 같아보이는데 사실은 껍데기만 제공하고 있는 것 같아요. `NavigateButton` 내에서 navigate 를 제공해주면 좋겠는데요. 
`Link` 컴포넌트를 확장하거나 내부에서 처리해 아래와 같이 사용하면 어떨까 생각이 드는데요! 자스민 의견이 궁금하빈다.
```suggestion
          <NavigateButton to="/shopping-cart">
```

Reply:
확실히 NavigateButton이라고 정해놓은것에 자체적으로 navigate기능을 주는것이 훨씬 좋은것 같네요! 수정하였습니다!